web/groups.go

```diff
@@ -197,14 +198,21 @@ func (s *QuickFeedService) getGroupUsers(request *qf.Group) ([]*qf.User, error)
 	return users, nil
 }
 
-// only allow letters, numbers, dash and underscore.
-var regexpNonAuthorizedChars = regexp.MustCompile("[^a-zA-Z0-9-_]")
+// only allow letters, numbers, underscore, dash and period.
+var regexpNonAuthorizedChars = regexp.MustCompile("[^a-zA-Z0-9_.-]")
```

Comment:
Even if GitHub allows period in the repo name, we don’t need to allow it. I prefer to disallow it.